### PR TITLE
close page added

### PR DESCRIPTION
**Pull request by kkrisstoff**
good_bye layout added

### CODE REVIEW

File: modules/UI/feedback/FeedbackWindow.js [hunk 1]
```diff
@@ -117,13 +117,11 @@ function onFeedbackSubmitted(Feedback) {
         Feedback.feedbackScore,
         message);
 
-    // TODO: make sendFeedback return true or false.
-    Feedback.submitted = true;
```
Comment (kkrisstoff) [inline modules/UI/feedback/FeedbackWindow.js]: it returns object {score, message} now

File: css/_redirect_page.scss [hunk 2]
```diff
@@ -1,12 +0,0 @@
-html, body {
-    width: 100%;
-    height:100%;
-    color: $defaultColor;
-    background: $defaultBackground;
-}
-
-.redirectPageMessage {
-    text-align: center;
```
Comment (damencho) [inline css/_redirect_page.scss]: redirectPageMessage class is used in authError.html

File: modules/UI/feedback/FeedbackWindow.js [hunk 3]
```diff
@@ -116,13 +116,11 @@ function onFeedbackSubmitted(Feedback) {
...
```
Comment (damencho) [inline modules/UI/feedback/FeedbackWindow.js]: Feedback.submitted is used in Feedback.isSubmitted.

File: modules/UI/feedback/FeedbackWindow.js [hunk 4]
```diff
@@ -143,9 +141,7 @@ export default class Dialog {
     constructor() {
         this.feedbackScore = -1;
         this.feedbackMessage = '';
-        this.submitted = false;
```
Comment (damencho) [inline modules/UI/feedback/FeedbackWindow.js]: Same here, FeedbackWindow.submitted is used.

File: css/_redirect_page.scss (hunk 2 above)
Comment (kkrisstoff) [inline css/_redirect_page.scss]: I changed this class in authError.html

File: modules/UI/feedback/FeedbackWindow.js (hunk 3 above)
Comment (kkrisstoff) [inline modules/UI/feedback/FeedbackWindow.js]: It's correct. Fixed

File: modules/UI/feedback/FeedbackWindow.js [hunk 5]
```diff
@@ -179,6 +175,9 @@ export default class Dialog {
     }
 
     onHide() {
-        this.onCloseCallback(this.feedbackScore, this.feedbackMessage);
+        this.onCloseCallback({
+            score: this.feedbackScore,
+            message: this.feedbackMessage
+        });
     }
```
Comment (damencho) [inline modules/UI/feedback/FeedbackWindow.js]: These parameters do not reach maybeRedirectToWelcomePage as expected by this PR. They are passed to the callback passed to Feedback.openFeedbackWindow, which is only called in UI.requestFeedbackOnHangup.

File: modules/UI/close_page/ClosePage.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+     */
+    render() {
+        const fb = this.feedback;
+        const $html = $(compiledTpl({
+            thankMsg: getThankMsg(fb),
+            hintMsg: getHintMsg()
+        }));
+
+        $('body').html(APP.translation.translateElement($html));
+    }
```
Comment (damencho) [inline modules/UI/close_page/ClosePage.js]: Replacing the html shows to the user the desired page, but the camera is left open and it feels weird.

**Comment by damencho** [comment]
We decided to close this PR for the time being and we will try another approach.

**Pull request closed without merging**